go/oasis-node/cmd/registry/node/node.go

```diff
@@ -291,28 +289,19 @@ func doList(cmd *cobra.Command, args []string) {
 	conn, client := doConnect(cmd)
 	defer conn.Close()
 
-	nodes, err := client.GetNodes(context.Background(), &grpcRegistry.NodesRequest{})
+	nodes, err := client.GetNodes(context.Background(), 0)
```

Comment:
Should we define a const for `current height` somewhere on the off-chance that we wish to stop using `0` here?

Reply:
Good idea, done, I hope I didn't miss a spot.